src/review-1-4-1-7-demo/Blog.jsx

```diff
@@ -55,16 +87,23 @@ const Blog = () => {
     return (
       <div>
         <h1>Users</h1>
-        <div className="users-list">
-          {users.map((user) => (
-            <div key={user.id}>
-              <strong>{user.name}</strong>
-              <p>Email: {user.email}</p>
-              <p>Website: {user.website}</p>
-              <button onClick={() => handleUserClick(user)}>View Posts</button>
-            </div>
-          ))}
-        </div>
+        {usersState.error !== null ? <>Oh no, something went wrong 😭</> : null}
```

Comment:
we conditionally render an error message if something went wrong with the API call